### PR TITLE
Noetic Migration and Bugfixes for OpenCV AdaBoosting

### PR DESCRIPTION
**Pull request by ipa-foj**
- Noetic migration of the code to run with OpenCV 4.2
- Addresses issue #12 and #14

### CODE REVIEW

File: ipa_room_segmentation/ros/launch/room_segmentation_action_server.launch [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 
 	<!--  -->
-	<node ns="room_segmentation" pkg="ipa_room_segmentation" type="room_segmentation_server" name="room_segmentation_server" output="screen" respawn="true" respawn_delay="2">
+	<node ns="room_segmentation" pkg="ipa_room_segmentation" type="room_segmentation_server" name="room_segmentation_server" output="screen" respawn="false" respawn_delay="2">
```
Comment (ipa-rmb) [inline ipa_room_segmentation/ros/launch/room_segmentation_action_server.launch]: Why false? Wasn't this our fallback for sporadic crashes of the component? Or does it never crash anymore?

**Review by ipa-rmb — DISMISSED** [review]
Very good, well done. Thanks a lot. Just answer my question.

File: ipa_room_segmentation/ros/launch/room_segmentation_action_server.launch (hunk 1 above)
Comment (ipa-foj) [reply ipa_room_segmentation/ros/launch/room_segmentation_action_server.launch]: True, overlooked that during the push, thanks for notifying. Reverted it to `respawn=true` in the new commit.

**Review by ipa-rmb — APPROVED** [review]
Thanks a lot for your bugfix!

**Pull request merged**